### ISSUE TITLE
Error building Carthage 0.26.0

### ISSUE THREAD

**Issue by yonaskolb** [lucianomarisi/JSONUtilities#26]
JSONUtilities doesn't build anymore in Carthage 0.26.0+
Not sure what the reason is. I've opened an issue there
https://github.com/Carthage/Carthage/issues/2230

**Comment by yonaskolb** [comment]
It looks like this is fixed in master after the Swift 4 upgrade. @lucianomarisi, any chance of getting a release out?

**Comment by lucianomarisi** [comment]
Absolutely, just released tag `5.0.0`. Will do a Cocoapods release later today.

**Comment by yonaskolb** [comment]
Awesome! Thanks @lucianomarisi! :)
